tests/net/ieee802154/l2/src/ieee802154_shell_test.c

```diff
@@ -25,7 +25,7 @@ extern struct net_pkt *current_pkt;
 extern struct k_sem driver_lock;
 extern uint8_t mock_ext_addr_be[8];
 
-static struct net_if *iface;
+static struct net_if *net_iface;
```

Comment:
Wow, this was a lot of work. Many thanks! The comment from Andy Ross (optionally) applies here as well: https://github.com/zephyrproject-rtos/zephyr/pull/61234#pullrequestreview-1566078112